src/test/java/nl/dtls/fairdatapoint/service/impl/FairMetaDataServiceImplTest.java

```diff
@@ -318,6 +318,21 @@ public void existenceMetric() throws
                 "http://www.example.com/cat1"));
         assertFalse(metadata.getMetrics().isEmpty());
     }
+    
+    /**
+     * Test existence of access rigghts
+     *
+     * @throws nl.dtls.fairdatapoint.service.FairMetadataServiceException
+     * @throws nl.dtl.fairmetadata4j.io.MetadataException
+     */
+    @DirtiesContext
```

Comment:
Do we need the `@DirtiesContext` for "read-only" tests? As long as we start the test under the assumption the context is clean/pristine, and the test doesn't affect the 'context', shouldn't we not have to worry about these things?